modules/azure-gaming/connected-resources/modules/resources.bicep

```diff
@@ -0,0 +1,157 @@
+@description('Deployment Location')
+param location string
+
+@description('Toggle to enable or disable zone redudance.')
+param isZoneRedundant bool = true
+
+@allowed([ 'new', 'existing', 'none' ])
+param newOrExistingStorageAccount string = 'none'
+param storageAccountName string = 'store${uniqueString(resourceGroup().id, subscription().id)}'
```

Comment:
The default value for storageAccountName is different in 3 places throughout the modules.

The CAF naming guidance suggests using st as the prefix. `st<project, app or service><###>`

Reply:
```suggestion
param storageAccountName string = 'st${uniqueString(resourceGroup().id, subscription().id)}'
```